supabase/functions/oauth/auth-url.ts

```diff
@@ -12,6 +14,11 @@ const generateUniqueRandomKey = () => {
     return String.fromCharCode.apply(null, array);
 };
 
+const base64UrlEncode = (str: string) =>
```

Comment:
add a comment that this is mapping a standard encoding to a web-safe encoding ? I thought it was actually supposed to base64 when i first read it.

Reply:
Yeah originally I had base64 encoding as part of this function but then took it. Renamed it and added comment 👍🏽